### DISCUSSION TITLE
Question: From which version of fwupd tool, Capsule-On-Disk has been made the default firmware update option ?

### DISCUSSION THREAD

**Discussion by peckyspecky** [fwupd/fwupd#7294] [General]
Hi,

I have tried to update the firmware using fwupd tool, but the system got rebooted to the OS and the firmware update did not happen. 
Checked and found that in Ubuntu 24.04 LTS which uses fwupd tool 1.9.16, Capsule-On-Disk mechanism is the default update option.
From which version did fwupd tool has made Capsule-On-Disk a default method to update firmware ?
Gone through the commits, but unable to find from which version did Capsule-on-Disk has been made as default to update the firmware ?

**Further details:**
OS:  Ubuntu 24.02 LTS
fwupd version: 1.9.16
Platform used: Intel x64 platform.
Should you need more information on this, please reply back.

Thanks,
Sathya

**Comment by superm1** [comment]
It's been enabled by default since fwupd 1.8.11.  There was a bug with Insyde on not handling CoD properly, and Insyde fixed that and it got enabled by default.  Here's the details: 

https://github.com/fwupd/fwupd/commit/2b96b9e93358f361c0003a290e1bc0ea3032631c

Who is the BIOS vendor for your system?

**Reply by hughsie** [reply]
Capsule-on-Disk is used whenever the OsIndicationsSupported EFI variable says it should be used.

**Reply by superm1** [reply]
> Thanks much!!!
> 
> We use AMI BIOS.

I don't believe we've seen any AMI systems using it yet, only Insyde so far.

I suggest you raise a bug with them, it might be an implementation issue.
